proposals/list-patterns-on-enumerables-2.md

```diff
@@ -0,0 +1,91 @@
+# List patterns on enumerables
+
+Champion issue: <https://github.com/dotnet/csharplang/issues/9005>
```

Comment:
Since the original proposal still exists, shouldn't we create a new issue instead?

Reply:
I'm not sure, happy to do so. @333fred is the choice of how to handle inactive proposals something that <https://github.com/dotnet/csharplang/blob/main/Design-Process.md> could speak to?

Reply:
It could, I suppose, but we definitely don't want to create a new issue here. The point is trying to keep historical context, we don't want to throw out existing context.